Modules/TRD/src/TrackletsTask.cxx

```diff
@@ -244,8 +244,19 @@ void TrackletsTask::buildHistograms()
   getObjectsManager()->startPublishing(mTrackletPositionRawn);
   mTrackletsPerEventn = new TH1F("trackletspereventn", "Number of Tracklets per event noise in;Tracklets in Events;Counts", 25000, 0, 25000);
   getObjectsManager()->startPublishing(mTrackletsPerEventn);
-  mTrackletsPerTimeFrame = new TH1F("trackletspertimeframe", "Number of Tracklets per timeframe;Tracklets in TimeFrame;Counts", 25000, 0, 500000);
+  constexpr int nLogBins = 100;
+  float xBins[nLogBins + 1];
+  float xBinLogMin = 0.f;
+  float xBinLogMax = 8.f;
+  float logBinWidth = (xBinValMax - xBinValMin) / nLogBins;
+  for (int iBin = 0; iBin <= nLogBins; ++iBin) {
+    xBins[iBin] = TMath::Power(10, xBinValMin + iBin * logBinWidth);
+  }
+  mTrackletsPerTimeFrame = new TH1F("trackletspertimeframe", "Number of Tracklets per timeframe;Tracklets in TimeFrame;Counts", nLogBins, xBins);
+  mTrackletsPerTimeFrame->SetStats(11111);
   getObjectsManager()->startPublishing(mTrackletsPerTimeFrame);
+  getObjectsManager()->setDefaultDrawOptions(mTrackletsPerTimeFrame->GetName(), "logx");
+  // getObjectsManager()->setDisplayHint(mTrackletsPerTimeFrame->GetName(), "SetOptStat(11111)"); // FIXME: how to show number of underflow bin entries by default?
```

Comment:
@knopers8 is it possible to show the underflow bin count be default in QCG?

Reply:
Hm, I don't know myself. The display hint and default draw options are passed by QCG to JSROOT. Is there something else needed to allow for it?

Reply:
Ok, no problem. I'll try to play around a bit and see if I can make it work in the QCG test.